build-scripts/generateRunners.ts

```diff
@@ -56,11 +56,11 @@ run().then((result) =>
 `
 
     const output = header + importStament + runStatement + run
-    fs.writeFileSync(`Tasks/${filename}/${filename}.ts`, output)
+    fs.writeFileSync(`Tasks/${filename}/${filename}.runner.ts`, output)
```

Comment:
Does the clean script need to account for these files?

Reply:
I mean sure but the clean script doesn't even work properly really. I'd much rather tackle that in the branch werlla/convertCleanToTs

Reply:
Ah actually now that I think about it, no, still because we just nuke the _build directory with clean.